src/components/Badge/Badge.tsx

```diff
@@ -1,14 +1,15 @@
-import React, { FunctionComponent } from 'react';
-
 import classnames from 'classnames';
+import React, { FunctionComponent } from 'react';
 
 import { DefaultProps } from '../../types';
 
+import { BadgeType } from './Badge.types';
+
 import './Badge.scss';
 
 export interface BadgeProps extends DefaultProps {
 	text: string;
-	type?: 'default' | 'success' | 'error';
```

Comment:
Wordt nergens anders gebruikt dus moet niet perse in een aparte type gezet worden